lib/UI/prev_workout/prev_exercise_card.dart

```diff
@@ -25,31 +18,39 @@ class PrevExerciseCard extends StatefulWidget {
     required this.isTemplate,
     required this.setList,
     required this.prevworkoutData,
-  });
+    Key? key,
+  }) : super(key: key);
+  final String name;
+  final Exercise exercise;
+  Function(Exercise) addExercise;
+  Function(Exercise) updateExisitingExercise;
+  final bool isTemplate;
+  List<PrevSetWidget> setList;
+  PrevWorkoutData prevworkoutData;
```

Comment:
also here and the rest of the places where functions don't have return types. It might indeed be better to do this in smaller PR's because this pr is already pretty massive.